functional-test/src/test/java/org/zanata/feature/language/JoinLanguageTeamTest.java

```diff
@@ -66,8 +66,9 @@ public void translatorJoinsLanguageTeam() throws Exception {
         assertThat(languagePage.getMemberUsernames())
                 .contains("translator")
                 .as("Translator is a listed member of the pl team");
-        assertThat(hasEmailRule.emailsArrivedWithinTimeout(1, 5,
+        assertThat(hasEmailRule.emailsArrivedWithinTimeout(1, 30,
```

Comment:
No, it's just a matter of being an external interface (mail) that may, occasionally, take a touch more than 5 seconds.  We wait 20-30 seconds for everything else, so I think this is ok.